.devcontainer/devcontainer.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ESPHome Dev",
-  "image": "esphome/esphome-lint:dev",
+  "image": "docker.io/esphome/esphome-lint:dev",
```

Please dont make unrelated changes in this pr
```suggestion
  "image": "esphome/esphome-lint:dev",
```